deps/cflag/cflag.h

```diff
@@ -40,13 +40,13 @@ struct cflag {
 
 
 #define CFLAG(_t, _name, _letter, _data, _help) \
-    ((struct cflag) {                           \
+    {                                           \
```

Comment:
I think this change should not be needed, because it results in valid C99 syntax. According to [the GCC C99 status page](https://gcc.gnu.org/c99status.html):

> C99 is substantially completely supported as of GCC 4.5 [...]

Have you tried using `make CFLAGS='-std=c99'` or `make CFLAGS='-std=gnu99'` instead of patching the source like this?

Reply:
I am using `-std=c99`. Unfortunately it looks like this syntax is only supported starting with GCC 5. I'm using Compiler Explorer to verify (https://godbolt.org/z/74rzrq).

One note, even with GCC 5 and later if you add the `-pedantic` flag this warning comes back which makes me wonder if this is valid C99 syntax?